src/api/CortexClient.ts

```diff
@@ -32,17 +33,12 @@ export class CortexClient implements CortexApi {
     this.discoveryApi = options.discoveryApi;
   }
 
-  async syncEntities(
-    entities: Entity[],
-    customMappings?: CustomMapping[],
-    teamOverrides?: TeamOverrides,
-    requestOptions?: RequestOptions,
-  ): Promise<void> {
+  async submitEntitySync(entities: Entity[], customMappings?: CustomMapping[], teamOverrides?: TeamOverrides, requestOptions?: RequestOptions): Promise<EntitySyncProgress> {
     const withCustomMappings: Entity[] = customMappings
       ? entities.map(entity => applyCustomMappings(entity, customMappings))
       : entities;
 
-    return await this.post(`/api/backstage/v1/entities`, {
+    return await this.post(`/api/backstage/v1/entities/sync`, {
```

Comment:
oh wait we wanna add the new allow multiple query param right?

Reply:
😨  oops